lib/new_relic/transaction/erlang_trace.ex

```diff
@@ -92,6 +96,18 @@ defmodule NewRelic.Transaction.ErlangTrace do
   defp trace_proc_lib_spawn_link do
     :erlang.trace_pattern({:proc_lib, :spawn_link, :_}, [{:_, [], [{:return_trace}]}], [])
     :erlang.trace_pattern({:proc_lib, :start_link, :_}, [{:_, [], [{:return_trace}]}], [])
+
+    # Starting with elixir 1.15 tasks are spawned with Task.Supervised.
+    # See https://github.com/elixir-lang/elixir/commit/ecdf68438160928f01769b3ed76e184ad451c9fe
```

Comment:
Great find! I think this was the thing I was looking for for quite some time. Thank you!